src/main/java/com/example/notes/entity/NoteEntity.java

```diff
@@ -10,6 +13,9 @@
 @Entity
 @Table(name = "notes")
 @Data
+@Builder
```

Comment:
Let us avoid using @Builder with entities for now.

Reply:
ok